.travis/Dockerfile

```diff
@@ -26,10 +26,10 @@ RUN cd /build && \
 	git clone -b $(curl -s -L http://grpc.io/release) https://github.com/grpc/grpc && \
 	cd /build/grpc && \
 	git submodule update --init && \
-	make -j && \
+	make -j4 && \
```

Comment:
I wouldn't worry about this Dockerfile though, since we can assume the Docker image would be built (by us) on a reasonably beefy machine.